src/Components/Api.js

```diff
@@ -34,4 +34,10 @@ const getComments = (article_id) => {
 })
 }
 
-export { getTopics, fetchArticlesByTopics, fetchArticleById, getComments }
\ No newline at end of file
+const patchVotes = (article_id, incVotes) => {
+  return api.patch(`/articles/${article_id}`, { inc_votes: incVotes} ).then(() => {
+    console.log('database updated')
+  })
+}
```

Comment:
This is great but I'd recommend removing the console log to clean up the code 🧹 ✨  and prevent the console getting jammed up with logs

Reply:
I'm debating with myself whether it would be considered best practice to return the updated article from your API function - even though you aren't using it in the `Votes` component as it's being done optimistically... because you might want to have access to that data if you reuse this API function elsewhere 🤔